package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Open source UI components and visual style guide for U.S. government websites",
   "main": "src/js/start.js",
   "scripts": {
-    "prepublish": "gulp build",
+    "prepublish": "bundle && gulp build",
```

Comment:
yeah I agree with this, but the `/docs` directory is still a Ruby project so the `Gemfile` being there is what makes it a ruby project. 0.2% is still something I guess.

<img width="985" alt="screen shot 2016-03-11 at 2 07 13 pm" src="https://cloud.githubusercontent.com/assets/706004/13712608/9f57fe30-e792-11e5-9585-5a566688438b.png">